pkg/acsengine/defaults.go

```diff
@@ -684,6 +684,14 @@ func setOrchestratorDefaults(cs *api.ContainerService) {
 			a.OrchestratorProfile.KubernetesConfig.UseInstanceMetadata = helpers.PointerToBool(api.DefaultUseInstanceMetadata)
 		}
 
+		if a.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == "" {
+			a.OrchestratorProfile.KubernetesConfig.LoadBalancerSku = api.DefaultLoadBalancerSku
+		}
+
+		if a.OrchestratorProfile.KubernetesConfig.ExcludeMasterFromStandardLB == nil {
+			a.OrchestratorProfile.KubernetesConfig.ExcludeMasterFromStandardLB = helpers.PointerToBool(api.DefaultExcludeMasterFromStandardLB)
```

Comment:
As @CecileRobertMichon suggested, let's convert the default value assignment of `ExcludeMasterFromStandardLB` to a function. The function should look like this:

- is this a 1.11 or greater cluster that is using "Standard" LB?
  - if so, set to true (based on comments from @feiskyer)
  - if not, false